### DISCUSSION TITLE
Extended attributes not available

### DISCUSSION THREAD

**Discussion by tsucaet** [MasaCMS/MasaCMS#153] [General]
When I add a new Class Extension with some extended attributes to the Base/Default section, the Extended Attributes tab in the content editor doesn't show up for any page type.
The same happens when installing the Masa Google Sitemap Plugin, the Extended Attributes tab with the Google Sitemap settings isn't available.

**Comment by brucebailey** [comment]
If I am reading that right, you are having the same issue I had. The extended attributes are showing under the Basic tab now rather than under the "Extended Attributes" tab.  All the way to the bottom, just before the "Summary" section you will see, in red, "Manage Extended Attributes".

**Reply by tsucaet** [reply]
Thanks for the clarification, there's indeed a 'Manage Extended Attributes' link available in each tab (Basic, Summary, Image,...), but the Manage Extended Attributes page itself is empty. 

I reinstalled the Masa Google Sitemap plugin (which actually creates a Class Extension with some attributes in the Base/Default section), but these attributes are not showing up in the Manage Extended Attributes page. 
For testing purposes, I created a Class Extension with some attributes in the Page/Default section and these attributes are actually shown in the Manage Extended Attributes page under the Basic tab.

It seems that the attributes that resides under the Base/default section are nowhere available whereas it should be available in any content type.

**Comment by guustnieuwenhuis** [comment]
@tsucaet Could you create an issue for this? We'll look into it.

Btw, in 7.4 there is a new setting `showextensionsindefault`. When set to true (default is false), the extended attributes for the basic tab are show in the big area on the left (below the body) instead of being behind a button. We've added this option based on feedback from one of the supporting companies.

**Reply by tsucaet** [reply]
Issue created, https://github.com/MasaCMS/MasaCMS/issues/159
Thanks for looking into this issue.

**Comment by kerryhill** [comment]
Can someone tell me why on some pages it shows the extended attributes and on some it does not?

Example:
![image](https://github.com/MasaCMS/MasaCMS/assets/35034737/6d38bf42-fb8b-4f00-9c59-699214a81365)

**Reply by guustnieuwenhuis** [reply]
Hi @kerryhill, I would need a bit more context and information to be able to understand what might be going on.
